org.eclipse.xtext.maven.plugin/src/test/resources/it/generate/pom.xml

```diff
@@ -18,18 +18,18 @@
 		<eclipse-text-version>3.12.300</eclipse-text-version>
 		<core-resources-version>3.18.200</core-resources-version>
 		<core-runtime-version>3.26.100</core-runtime-version>
-		<emf-common-version>2.24.0</emf-common-version>
-		<emf-ecore-version>2.26.0</emf-ecore-version>
-		<emf-edit-version>2.17.0</emf-edit-version>
-		<emf-version>2.26.0</emf-version>
-		<emf-xmi-version>2.16.0</emf-xmi-version>
-		<emf-codegen-version>2.22.0</emf-codegen-version>
-		<emf-codegen-ecore-version>2.29.0</emf-codegen-ecore-version>
+		<emf-common-version>2.30.0</emf-common-version>
```

Comment:
I just noted this in the parent POM of the IT projects.
All these versions are useless because the parent import the dev BOM, so the versions are taken from there.
Maybe I can adjust them in another issue...

Reply:
Thank you for filing the follow-up ticket, @LorenzoBettini